doc/02-Installation.md

```diff
@@ -201,6 +202,80 @@ After creating the database, you can import the Icinga DB schema using the follo
 mysql -u root -p icingadb </usr/share/icingadb/schema/mysql/schema.sql
 ```
 
+### Configuring PostgreSQL <a id="configuring-icingadb-pgsql"></a>
```

Comment:
I have no doubt Postgressers will find their way through the docs.

Reply:
No doubt that there are users that will get it right. But if someone just follows the instruction from top to bottom, they might end up setting up both. So why not group both sections in a new section that starts with an instruction that states these are alternatives and you have to set up just one, probably even with a suggestion which one to choose if you don't know yourself.

Reply:
> But if someone just follows the instruction from top to bottom, they might end up setting up both.

OK.. so you think.. once someone has set up MySQL and hit the setup Postgres headline.. they'd not consider it strange to setup both?

Reply:
You're assuming that everyone installing Icinga DB is familiar with Redis, MySQL, MariaDB, and PostgreSQL. The only hint so far that you need Redis and one of the remaining three is that at the top of the document, there is one bullet point for Redis and one for MySQL/MariaDB/PostgreSQL.

So I think this would a rather simple change for us that improves the documentation so that it could save someone from pointlessly setting up multiple RDMBS.

Reply:
Ah, wait! Isn’t https://github.com/Icinga/icingadb/pull/221/files/a357ed9516febb6c99230f2efca21ff03a736bac#diff-a8d6a4bf6bf9e106f20ce651f485c61a9884fe3e30fbd30598e631aa33a3e66bR6-R7 enough?

Reply:
I support Julian's thoughts here.

Reply:
I think it makes sense to categorize it like this:

### Setting Up the Database

Small explanation that either a MySQL/MariaDB or a PostgreSQL database needs to be created. It can also be useful to explain the function of the database, i.e. that is the data source for Icinga Web 2, reporting and so on.

#### MySQL/MariaDB

#### PostgreSQL

Reply:
OK, for me the latter is **definitively** another construction area. If we need this, we need this independent of Postgres support. (The small explanation is already there.)

Reply:
Since I choose to introduce the Database Setup chapter in this PR, it only makes sense to add a little explanation as well.

Reply:
This change is still missing.

Reply:
I consider this PR is already large enough. Why don’t we improve the DB docs separately and I'll rebase later on? Also you seem to understand far better than me what/how/why to do on that.